src/explore-education-statistics-admin/src/services/publicationService.ts

```diff
@@ -117,8 +138,13 @@ const publicationService = {
     return client.put(`/publications/${publicationId}`, publication);
   },
 
-  getPublication(publicationId: string): Promise<Publication> {
-    return client.get<Publication>(`/publications/${publicationId}`);
+  getPublication(
+    publicationId: string,
+    permissions = false,
```

Comment:
Again, I'd suggest making the name of this variable a bit more explanatory - `includePermissions` or summat.